ng2-components/ng2-alfresco-core/src/services/alfresco-translation.service.ts

```diff
@@ -22,24 +22,38 @@ import { AlfrescoTranslateLoader } from './alfresco-translate-loader.service';
 
 @Injectable()
 export class AlfrescoTranslationService {
+    defaultLang: string = 'en';
     userLang: string = 'en';
     customLoader: AlfrescoTranslateLoader;
 
     constructor(public translate: TranslateService) {
-        this.userLang = translate.getBrowserLang() || 'en';
-        translate.setDefaultLang(this.userLang);
+        this.userLang = translate.getBrowserLang() || this.defaultLang;
+
+        // this language will be used as a fallback when a translation isn't found in the current language
```

Comment:
can we enclose these line of code in a function that says what the comment says? and remove the comment?